src/HanziWriter.js

```diff
@@ -98,6 +99,7 @@ HanziWriter.prototype.setOptions = function(options) {
   });
   this._highlightCharOptions = assign({}, this._mainCharOptions, {
     strokeColor: this._options.highlightColor,
+    highlightCompleteColor: this._options.highlightCompleteColor,
```

Comment:
It doesn't look like this is used by the `CharacterRenderer` class - no need for this line

Reply:
Oh, good catch. Sorry for that.